tests/5.0/allocate/test_allocate_on_device.c

```diff
@@ -0,0 +1,57 @@
+//===----------------- test_allocate_on_device.c ------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// Tests the allocate directive with allocator clause in the target region.
+// A variable 'x' is allocated using and omp_default_mem_alloc allocator. 
```

Comment:
This test fails with latest llvm trunk on summit due to 
`nvlink error   : Undefined reference to 'omp_alloc' in '/tmp/test_allocate_on_device-0fac84.cubin'
nvlink error   : Undefined reference to 'omp_free' in '/tmp/test_allocate_on_device-0fac84.cubin'`